README.md

```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+![](./src/img/todo.jpg)
 
-> Description the project.
+# To Do App
 
+> This is an Educational project aimed at teaching how to give and receive professional code review. This exercise was carried out between my self and a single partner who reviewed my code base for different issues using the HTML/CSS best practices, Modern JavaScript best practices, KISS, YAGNI, and DRY guidelines.
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+## Live URL
+[Deployment](https://acedavon.github.io/todo-list/)
 
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML, CSS, JavaScript
+- Git, GitHub, Webpack, Webpack-cli, webpack-dev-server
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+**Setting this project Locally,**
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+1. Clone the Repo or Download the Zip file
+2. Run the live server or simply open index.html with your browser.
 
-### Install
+## Project Mockup
 
-### Usage
-
-### Run tests
-
-### Deployment
+- All views
+  ![](./src/img/3-devices-black.png)
 
+- Desktop
+  ![](<./src/img/desktop%20(1).png>)
 
+- Mobile
+  ![](./src/img/mobile-black.png)
 
```

Comment:
I guess you may have mistakenly skipped adding the steps at which a user can install, use, run test, and deploy your web app. These are usually done by adding some simple code snippets that the user will run locally on their machine.

What do you think about implementing this suggestion? 💭 It is very important for your README file to have full details on how to use your web app.